qemu/tests/live_snapshot_basic.py

```diff
@@ -1,35 +1,39 @@
 import logging
 
-from virttest import utils_test
 from virttest import utils_misc
-from virttest import storage
-from virttest import error_context
 from virttest import data_dir
 from virttest.qemu_storage import QemuImg
 from avocado.core import exceptions
+from qemu.tests import block_copy
 
 
-class LiveSnapshotBase(object):
+class LiveSnapshot(block_copy.BlockCopy):
 
     """
     Provide basic functions for live snapshot test cases.
     """
 
-    def __init__(self, params, env):
+    def __init__(self, test, params, env, tag):
         """
         Init the default values for live snapshot object.
 
         :param params: A dict containing VM preprocessing parameters.
         :param env: The environment (a dict-like object).
         """
-        self.params = params
-        self.env = env
-        self.vm = env.get_vm(params["main_vm"])
-        self.snapshot_file = params.get("snapshot_file")
-        self.snapshot_format = params.get("snapshot_format", "qcow2")
-        self.snapshot_mode = params.get("snapshot_mode", "absolute-paths")
-        self.node_name = params.get("node_name")
-        self.snapshot_node_name = params.get("snapshot_node_name")
+        super(LiveSnapshot, self).__init__(test, params, env, tag)
+
+    def parser_test_args(self):
+        self.default_params = {"login_timeout": 360}
+        self.snapshot_file = self.params.get("snapshot_file")
+        self.node_name = self.params.get("node_name")
+        self.snapshot_node_name = self.params.get("snapshot_node_name")
+        self.snapshot_args = {"mode": "absolute-paths",
```

Comment:
above lines should be in __init__ function, another question is that  where to update default mode value?